### ISSUE TITLE
Test results using pretrained model are not as good as what you posted in the paper

### ISSUE THREAD

**Issue by cfanyyx** [TianxiangMa/MUST-GAN#4]
![image](https://user-images.githubusercontent.com/7723243/127610966-6fc5d640-193b-413e-8a46-6b04a2286cac.png)
![image](https://user-images.githubusercontent.com/7723243/127611160-b58375ae-0c18-46d2-9a9c-f152e4a0e5b0.png)
![image](https://user-images.githubusercontent.com/7723243/127611275-a730d181-665d-42a9-91d3-e5c3e4fbcc2c.png)

**Comment by TianxiangMa** [comment]
In fact for cases where there is a large difference between the pose of the source image and the target pose, such as half-body to full-body, the information of the input image is difficult to meet the need for accurate generation of the target image. 

While for supervised methods (trained using paired data) the network can remember this pattern of image generation under large pose differences, for the self-supervised method proposed in this paper, we do not force to remember this transformation relationship, but learn spontaneously the decoupling and re-fusion of appearance and pose, thus requiring the information embedded in the input image to satisfy the need to generate the target image.

This is the reason why our method is lower than the supervised SOTA methods in calculating quantitative metrics. In addition, we consider that the currently commonly used Deepfashion paired test set is flawed, i.e., there are a large number of data pairs where the whole body is generated from the half body. And for a fair comparison we still use the test data pairs sampled by most of the current methods.

Some test results of our pretrained model:
![image](https://user-images.githubusercontent.com/45935929/127823156-9af4b6c6-4b04-4cc3-8ebb-da9d01bd88f4.png)
![image](https://user-images.githubusercontent.com/45935929/127823665-f45ac4c3-802e-4a73-a44e-48090885365c.png)
![image](https://user-images.githubusercontent.com/45935929/127823818-498e1da7-2e58-4b36-9272-a6492bc411e4.png)
![image](https://user-images.githubusercontent.com/45935929/127828461-2d2a084a-532a-42bc-a434-c69b98c4762d.png)
![image](https://user-images.githubusercontent.com/45935929/127829601-c10459d8-d3fe-403c-95c6-bce392e979d9.png)

**Comment by rouge012** [comment]
Hi! How can I solve the error "on.close() takes 1 positional arguments but 3 was given" when test?How can I properly use the pre-trained model for testing?
